### DISCUSSION TITLE
Option to hide / collapse the tool tray

### DISCUSSION THREAD

**Discussion by davidolrik** [zsviczian/obsidian-excalidraw-plugin#634] [Ideas]
I use Obsidian on an iPad mini and find that the tool tray often obscures the place I want to draw when using the draw tool.

I would love a way to hide or collapse the tool tray so I can see and use the full screen.

**Comment by YesterKo** [comment]
I don't if it's exactly what you want, but it's possible to enable zen mode, which hides most other panels, leaving only the top tool selector.

You can enter zen mode by:
1. right clicking/ holding down on empty space
2. selecting zen mode
3. Wohoo! You're in zen mode!

Exit the same way or clicking on the button in the lower-right corner.

Hope it is a suitable solution for you!

**Reply by davidolrik** [reply]
This is exactly what I was looking for! - Awesome!

**Reply by YesterKo** [reply]
Glad to be of help! :)
